packages/devextreme/scss/widgets/generic/treeList/_index.scss

```diff
@@ -133,6 +138,15 @@ $treelist-icon-container-padding-right: $generic-grid-base-checkbox-icon-size +
     left: $treelist-checkbox-offset-left;
     margin-top: $treelist-select-all-checkbox-margin-top;
   }
+
+  .dx-treelist-checkbox-size::before {
+    content: '';
+    width: $treelist-select-all-interaction-area-size;
+    height: $treelist-select-all-interaction-area-size;
+    position: absolute;
+    left: math.div($generic-grid-base-checkbox-icon-size - $treelist-select-all-interaction-area-size, 2);
```

Comment:
Let's create a new variable instead of the calculations: $generic-grid-base-checkbox-size-left and $generic-grid-base-checkbox-size-top.